Create Api Docs page

mcapeta opened the pull request:
Creating API documentation according to OpenAPI Specification v3.0.1.

mcapeta committed:
Create api docs definition

mcapeta committed:
Create api docs router

mcapeta committed:
Add docs path to dev server proxy table

mcapeta committed:
Add docs route to VueRouter

- redirect request to matching server side request

mcapeta committed:
Move docs to iframe element

- ditch redirection flow
- define new route with embedded docs

mcapeta committed:
Rename method

- it's a frame body, not form body

vladimyr committed:
Refactor documentation view

- swapping yaml parser with faster & safer one
- adding loading transition on documentation iframe
- using named route instead of path

vladimyr committed:
Enable console api overriding :wrench:

- ignoring error log

vladimyr reviewed (commented):
This looks good 👍 
Did some minor changes/refactors. Looks good codewise but I need to take second pass through actual api spec :bowtie:

vladimyr committed:
Implement generic SwaggerViewer component :tada:

- preauthorizing swagger client using received configuration

vladimyr committed:
Clenaup :lipstick:

vladimyr committed:
Adjust fade-in duration :lipstick:

vladimyr committed:
Hide redundant parts of SwaggerUI :lipstick:

mcapeta committed:
Redefine docs api endpoint

- ditch `swagger-ui-express`
- expose docs in `.json`, `.js` and `.yaml` format

mcapeta committed:
Add api docs definition to client app

mcapeta committed:
Create User store getters

mcapeta committed:
Add `profile` property to Integration

mcapeta committed:
Define `name` property getter :bug:

Resolve property value if query was built through the `User` model

mcapeta committed:
Change User list filters

 - switch single role filter to multiple
 - redefine dependant component

mcapeta committed:
Change docs UI

- ditch iframe approach
- authorize requests with integration users
